### ISSUE TITLE
Last line of CSV makes node crash?

### ISSUE THREAD

**Issue by Extarys** [signicode/scramjet#18]
It's me again.

After playing with multiple CSV files, it seems that the last line doesn't get inserted.

I'm outputing the line in the `.each` loop.
On a CSV file containing the list of countries, all values of the invisible last line are undefined or NaN.
```
arg: [ 'BVT', 'BV', '74', 'Bouvet Island', '55', '', null, null, '', 'AN' ]
arg: [ undefined,
  undefined,
  undefined,
  '',
  undefined,
  undefined,
  NaN,
  NaN,
  undefined,
  undefined ]
```
I opened the file, there is no last line (the code was published in another issue here). I receive 'Read entire file' at the end since it didn't crash.

On a CSV file containing provider's rate list, the last line is properly interpreted in the `.each` loop but node crashes.

```
arg: [ 'standard',
  'Singapore - Singapore - National - Voice (with CLI)',
  '2018-10-04',
  60,
  0.506,
  2,
  0,
  60,
  0,
  2,
  50,
  30,
  0.209,
  'SGP' ]
ℹ info Inserted   standard,Singapore - Singapore - National - Voice (with CLI),2018-10-04,60,0.506,2,0,60,0,2,50,30,0.209,SGP
Country found for Portugal PRT
arg: [ 'standard',
  'Portugal - Portugal - Local - Voice',
  '2018-10-25',
  60,
  0.0285,
  2,
  0,
  60,
  0,
  2,
  4.75,
  0,
  0.03,
  'PRT' ]
[nodemon] app crashed - waiting for file changes before starting...
```

I'm using the same code for both CSV file, exept the each loop where I treat each file differently because of different columns location. I'll try with a third CSV file.

**Comment by Extarys** [comment]
Related to #15 if I don't use shift I go to line 291 and then node crashes (File have 292 lines)
When using shift like you described, I can only go to line 239 then node crashes.

**Comment by MichalCz** [comment]
@Extarys how about we try to create a test repo for your case? Can you push a sample of your file and the code (just the breaking part) there? It'd be easier to pin-point the issue there.

I'm still feeling that there should be some message on what caused the crash.

**Comment by MichalCz** [comment]
@Extarys, please install `scramjet=^4.18.17`. We may see some error messages that may have been previously muted.
